### ISSUE TITLE
Java script error occured

### ISSUE THREAD

**Issue by helderussa** [mchristopher/PokemonGo-DesktopMap#1027]
This isn't working for me. Already tested with latest version and version 0.2.1 and before it loads the map is giving the following error and then a Java script error window. Then it loads the map properly but nothing happens (empty map only with the pinpoint, no pokemons). I've only use PTC account, created two new accounts but the issue persists. As a note on last version it says "Possible Ban" on Waitting button.

![image](https://cloud.githubusercontent.com/assets/21201178/17898825/1003c638-6951-11e6-8402-3c08c5a6ae67.png)

Any tip?

**Comment by drippybutt** [comment]
I usually ignore the errors and the map works fine. I just got mine working again, just make sure you do the following steps:

1. Delete the pokemon-go-map folder in C:users/username/AppData/Roaming (I had to show hidden files to be able to see this)

2. Create a new PTC account and login and accept TOS and catch your first pokemon and finish the initial tutorial

3. Get a new Google API key, my old one no longer works as of today

4. Ignore all the java errors, just click ok, and it should work just fine!
